README.md

```diff
@@ -0,0 +1,89 @@
+# zfs-replicate
+
+https://github.com/alunduil/zfs-replicate
+
+By Alex Brandt <alunduil@gmail.com>
+
+## Description
+
+zfs-replicate sends all ZFS snapshots to a remote host via SSH.  zfs-replicate
+does **not** create ZFS snapshots.
+
+zfs-replicate is based on [autorepl.py] used by [FreeNAS].
+
+zfs-replicate is related to several other projects which fit other niches:
+
+1. [sanoid]: A full snapshot management system.  Its companion application,
+   syncoid, handles replication with many available options.
+1. [zfs-replicate (BASH)]: A very similar project.  The major differences are
+   configuration style (our project uses parameters whereas this project uses a
+   BASH script), and the system expectations (e.g., logging controls).
+1. [znapzend]: Another scheduling and replicating system.
+1. [zrep]: A SH script with several control commands for snapshot replication.
+
+## Terms of Use
+
+You are free to copy, modify, and distribute zfs-replicate with attribution
+under the terms of the BSD-2-Clause licence.  See the LICENSE for details.
+
+## Prerequisites
+
+1. A remote system with a ZFS filesystem and the zfs CLI tools
+1. SSH access to that remote system
+1. If you're not using the root user remotely:
+   1. Ensure the user can mount filesystems
+      1. [FreeBSD]: `sysctl -w vfs.usermount=1`
+   1. Add ZFS permissions
+      1. Command: `zfs allow ${USER} ${PERMISSIONS} ${BACKUP_DATASET}`
+      1. Permissions
+         1. clone
+         1. create
+         1. destroy
+         1. hold
+         1. mount
+         1. promote
+         1. quota
+         1. readonly
+         1. receive
+         1. rename
+         1. reservation
+         1. rollback
+         1. send
+         1. snapshot
+1. A local ZFS filesystem and zfs CLI tools
+
+_N.B., don't use the root user to access your remote system._
+
+## How to use zfs-replicate
+
+1. `poetry install`
```

Comment:
I don't think so since the pyproject.toml file specifies what versions are acceptable so you'll get an error that clearly states that the version is incorrect.

Reply:
I don't think the pyproject.toml specifies exactly which version of poetry. unless you're able to look at the toml file and see what features of poetry you're using. 

for example, you're using the dev groups, which i think is a 1.2 (or later) feature. 
if i'm on an older version of poetry, i'll probably get some runtime error.
```python
RuntimeError

  The Poetry configuration is invalid:
    - Additional properties are not allowed ('group' was unexpected)
 ```

i do see on poetry, that the earliest version you can install is 1.2. so maybe this doesn't matter too much.

Reply:
My understanding is the new poetry core has those features. I'm inclined to wait for an issue report on that one. If you agree mark this resolved otherwise I'll add a note when I get home.